simplicity-sys/depend/wrapper.c

```diff
@@ -0,0 +1,38 @@
+#ifndef WRAPPERS_H
+#define WRAPPERS_H
+#include "simplicity/frame.h"
+
+const size_t c_sizeof_UWORD = sizeof(UWORD);
+const size_t c_sizeof_frameItem = sizeof(frameItem);
+
+void c_initReadFrame(frameItem *frame, size_t n, UWORD *from)
+{
+    *frame = initReadFrame(n, from);
+}
+
+void c_initWriteFrame(frameItem *frame, size_t n, UWORD *from)
+{
+    *frame = initWriteFrame(n, from);
+}
+
+bool c_readBit(frameItem *frame)
+{
+    return readBit(frame);
+}
+
+void c_writeBit(frameItem *frame, bool bit)
```

Comment:
In 7cea6a1dcd9432114558a55758fc985b3d45413c:

I could go either way on this -- the only difference between the wrapped function and the unwrapped one is the presence of `static`. Arguably we should change upstream to expose these, or use a patch-applying trick like we do in rust-secp. (But on the other hand, applying patches is ugly and sometimes breaks, so..).

At the very least we should add a comment explaining that we're doing this since the internal function is `static`. On the others we should mention that we're also doing it to avoid passing C structures by value, since we're unsure about the FFI story there.

Reply:
The inlining was also an issue here. Inline functions don't have symbols to refer to.

Reply:
Makes sense. I think we should have a block comment in this file explaining this.

Reply:
The new commit looks fine, though you wrote `weadBit` rather than `writeBit`.